apps/mobile/src/features/send/send-form/hooks/use-send-form-btc.tsx

```diff
@@ -93,7 +94,11 @@ export function useSendFormBtc() {
 
     onInitSendTransfer({ utxos, feeRates }: SendFormBtcContext, values: SendFormBtcSchema) {
       try {
-        const { recipient: recipientAddress, feeRate } = values;
+        const { amount: inputAmount, recipient: recipientAddress, feeRate } = values;
+        if (!isValidPrecision(+inputAmount, BTC_DECIMALS)) {
```

Comment:
@tigranpetrossian I am just adding this check here for now to minimise disruption but as @kyranjamie rightly pointed out, this precision check should be in the form schema checks for `amount`. 

I excluded it from the main transaction validation code as it needs to use the raw input amount

Reply:
@pete-watters Thanks, noted! Will integrate into the schema.